doc/nrf/releases/release-notes-latest.rst

```diff
@@ -20,792 +24,40 @@ The following sections provide detailed lists of changes by component.
-    See the ``CONFIG_MCUBOOT_CLEANUP_ARM_CORE`` option.
-  * Allowed the final data chunk in the image to be unaligned in the serial-recovery protocol.
-  * Updated the ``CONFIG_BOOT_DIRECT_XIP_REVERT`` option to be valid only in xip-mode.
-  * Added an offset parameter to the tinycrypt ctr mode so that it can be properly used as a streaming cipher.
-  * Configured the bootloader to use a minimal CBPRINTF (:option:`CONFIG_CBPRINTF_NANO`) implementation.
-  * Configured logging to use :option:`CONFIG_LOG_MINIMAL` by default.
-  * Fixed a vulnerability with nokogiri<=1.11.0.rc4.
-  * Introduced a bootutil_public library that contains code common to MCUboot and the DFU application.
-    See :option:`CONFIG_MCUBOOT_BOOTUTIL_LIB`.
-
-* Image tool:
-
-  * Updated the tool to print an image digest during verification.
-  * Added a possibility to set a confirm flag for HEX files as well.
-  * Updated the usage of ``--confirm`` to imply ``--pad``.
-  * Fixed the argument handling of ``custom_tlvs``.
-  * Added support for setting a fixed ROM address in the image header.
-
+There are no entries for this section yet.
 
 Mcumgr
 ======
 
-The mcumgr library contains all commits from the upstream mcumgr repository up to and including snapshot ``74e77ad08``.
-
-The following list summarizes the most important changes inherited from upstream mcumgr:
-
-* Fixed an issue with devices running MCUboot v1.6.0 or earlier where a power outage during erase of a corrupted image in slot 1 could result in the device not being able to boot.
-  In this case, it was not possible to update the device and mcumgr would return error code 6 (``MGMT_ERR_EBADSTATE``).
-* Added support for invoking shell commands (shell management) from the mcumgr command line.
-* Added optional verification of an uploaded direct-xip binary, which will reject any binary that cannot boot from the base address of the offered upload slot.
-  This verification can be enabled through :option:`CONFIG_IMG_MGMT_REJECT_DIRECT_XIP_MISMATCHED_SLOT`.
-
+There are no entries for this section yet.
 
 Zephyr
 ======
 
-.. NOTE TO MAINTAINERS: The latest Zephyr commit appears in multiple places; make sure you update them all.
-
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``ff720cd9b343``, plus some |NCS| specific additions.
-
-For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
-
-.. code-block:: none
-
-   git log --oneline ff720cd9b343 ^v2.4.0-ncs1
-
-For a complete list of |NCS| specific commits, run:
-
-.. code-block:: none
-
-   git log --oneline manifest-rev ^ff720cd9b343
-
-The current |NCS| release is based on Zephyr v2.4.99.
-
-The following list summarizes the most important changes inherited from upstream Zephyr:
```

Comment:
> @mbolivar-nordic Can you suggest changes so that this content is correct after the upcoming NCS 1.5 release? The commit is probably still the same (?), but the command for checking must surely be updated.

The upstream commit is still the same, yes. The content for checking upstream commits should be updated to:

```
For a complete list of upstream Zephyr commits incorporated into |NCS| since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):

.. code-block:: none

   git log --oneline ff720cd9b343 ^v2.4.99-ncs1
```

It will print nothing, but that will change when `ff720cd9b343` changes after the next upmerge.

No other changes to this section.